mongo-helm/templates/mongo-secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: mongodb-secret
+type: Opaque
+data:
+    mongo-root-username: dXNlcm5hbWU=
+    mongo-root-password: cGFzc3dvcmQ=

```

Comment:
Please make it so that users can change the username and password from values file. If they do not specify then a default username and password is used.

Reply:
Fixed.